src/com/company/Contribuabil.java

```diff
@@ -1,28 +1,21 @@
 package com.company;
 
 public class Contribuabil {
-    String nume;
-    String cnp;
-    Proprietati[] proprietati=new Proprietati[10];
+    private String nume;
+    private String cnp;
+    private Proprietati[] proprietati = new Proprietati[10];
 
     public Contribuabil(String nume, String cnp, Proprietati[] proprietati) {
         this.nume = nume;
         this.cnp = cnp;
         this.proprietati = proprietati;
     }
 
```

Comment:
Creează metoda to string care să itereze prin proprietăți și pentru fiecare proprietate concatenează mesajul apeland metodele toString pe care le-ai suprascris în fiecare subclasă a superclasei Proprietati.